source/blog/2015-06-12-ember-1-13-0-released.md

```diff
@@ -0,0 +1,578 @@
+2.0 will have few new features. Among them are:
+
+#### each-in helper
+
+The `each-in` helper allows the iteration of object properties. For example,
+given this value for `items`:
+
+```js
+let items = {
+  "Item 1": 1234,
+  "Item 2": 3456
+};
+```
+
+The following template will iterate the keys:
+
+```hbs
+{{#each-in items as |key value|}}
+  <p>{{key}}: {{value}}</p>
+{{/each-in}}
+```
+
+Note that this helper is unbound. Adding a new property to `items` will not
+cause a rerender.
+
+Thanks to [@tomdale](http://twitter.com/tomdale) and implementing this
+feature, and to several others for helping push it to completion.
+
+#### get helper
+
+The `get` helper provides a bound way to fetch a single property from an object.
+For example given these items:
+
+```js
+let items = {
+  "Item 1": 1234,
+  "Item 2": 3456
+};
+```
+
+The following template display `1234`:
+
+```hbs
+{{get items 'Item 1'}}
+```
+
+This becomes more powerful when the second argument is a bound path:
+
+```hbs
+{{get items somePathWithAKey}}
+```
+
+Thanks to [@jmurphyau](https://twitter.com/jmurphyau) for implementing this feature.
+
+#### Notable Breaking Changes in Ember 2.0
+
+Ember 2.0 will remove a number of public APIs, all of which should have been
+deprecated in the 1.13 release and have a viable migration path. The Ember
+[Deprecation Guide](http://emberjs.com/deprecations/v1.x/) should provide a
+clear migration path for commonly used APIs.
+
+During the 2.0 beta cycle we will be removing and disabling  already
+deprecated APIs. Much of this work has not yet started, but the following
+represents what we believe the breaking changes will be.
+
+Many controller APIs are removed in Ember 2.0. Routeable controller still
+exist, but all other uses should have been deprecated. This includes:
+
+* `{{render "some-controller"}}`
+* `{{each item itemController="some-controller"}}` - This usage can be replaced
+  by nesting a component inside the item, and by using helpers.
+* `Ember.ObjectController`
+* `Ember.ArrayController`
+* The `{{controller}}` keyword
+* `needs:` on controllers
+
+All view APIs are removed in Ember 2.0. This includes:
+
+* `Ember.CoreView`, `Ember.View`, `Ember.ContainerView` and `Ember.CollectionView`
+* `Ember._Metamorph`, `Ember._MetamorphView`
+* The `{{view "some-view"}}` helper
+* The `{{view}}` keyword
+* `{{each itemView=`, `{{each itemViewClass=`, `{{each tagName=`, `{{each emptyView=`, `{{each emptyViewClass`
+* `Ember.Select` and `{{view "select"}}`
+* `Ember.Checkbox` is not removed, but will become a component instead of a view
+
+All Handlebars APIs are removed in Ember 2.0. This includes:
+
+* `Ember.Handlebars.helper`, `Ember.Handlebars.makeBoundHelper` and `Ember.Handlebars.helper`
+* `Ember.Handlebars.compile`
+
+Several template helpers are removed in Ember 2.0. These include:
+
+* `{{bindAttr}}`
+* `{{bind-attr}}`
+* `{{bind}}`
+* `{{template}}`
+* `{{linkTo}}`
```

Comment:
Might be worth saying that `{{#link-to}}` is not deprecated to avoid confusion. Something like `{{linkTo}} (use `{{link-to}}` instead)`